ivy/functional/ivy/data_type.py

```diff
@@ -1142,22 +1142,31 @@ def default_int_dtype(
     int_dtype: Optional[Union[ivy.IntDtype, ivy.NativeDtype]] = None,
     as_native: bool = False,
 ) -> Union[ivy.IntDtype, ivy.NativeDtype]:
-    """Summary.
-
+    """
     Parameters
     ----------
     input
-         (Default value = None)
+       (Default value = None) Number or array for inferring default int dtype.
     int_dtype
-
+       (Default value = None) Uint dtype to be returned as default.
     as_native
-         (Default value = None)
-
+       (Default value = None) Whether to return the default int dtype as native dtype.
     Returns
     -------
-        Return the input int dtype if provided, otherwise return the global default int
-        dtype.
-
+        Return the input int dtype if provided, otherwise return the global default
+        int dtype.
+    Examples
+    --------
+    >>> ivy.set_default_int_dtype(ivy.intDtype("int16"))
+    >>> ivy.default_int_dtype()
+    'int16'
```

Comment:
Need to have a blank line between examples to satisfy docstring style requirements